edgelet/aziot-edged/src/lib.rs

```diff
@@ -277,7 +277,7 @@ where
                             .unwrap_or(TRUST_BUNDLE_ALIAS)
                             .to_string(),
                         settings
-                            .manifest_trust_bundle_cert()
+                            .trust_bundle_cert()
```

Comment:
Is this correct?

Reply:
No it is not, I am not sure how that got in. It was not in previous PR: https://github.com/Azure/iotedge/commit/5e93f5834d51c404d82b6ec1e9213b611b725d84